Cargo.lock

```diff
@@ -818,9 +830,9 @@ dependencies = [
 
 [[package]]
 name = "glutin"
-version = "0.30.9"
+version = "0.30.10"
```

Comment:
I've updated it here https://github.com/alacritty/alacritty/pull/7079/files , but it's till in review...